### PR TITLE
Fixed typos in changelog_desktop.md

### PR DESCRIPTION
**Pull request by MalikIdreesHasanKhan**
Resolves 
Typos in changelog_desktop.md file. 
## Submitter Checklist:

- [ ] I confirm that no security/privacy review [is needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or that I have [requested](https://github.com/brave/security/issues/new/choose) one
- [ ] There is a [ticket](https://github.com/brave/brave-browser/issues) for my issue
- [ ] Used Github [auto-closing keywords](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) in the PR description above
- [ ] Wrote a good [PR/commit description](https://google.github.io/eng-practices/review/developer/cl-descriptions.html)
- [ ] Added appropriate labels (`QA/Yes` or `QA/No`; `release-notes/include` or `release-notes/exclude`; `OS/...`) to the associated issue
- [ ] Checked the PR locally: `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests`, `npm run lint`, `npm run gn_check`, `npm run tslint`
- [ ] Ran `git rebase master` (if needed)

## Reviewer Checklist:

- [ ] A security review [is not needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or a link to one is included in the PR description
- [ ] New files have MPL-2.0 license header
- [ ] Adequate test coverage exists to prevent regressions
- [ ] Major classes, functions and non-trivial code blocks are well-commented
- [ ] Changes in component dependencies are properly reflected in `gn`
- [ ] Code follows the [style guide](https://chromium.googlesource.com/chromium/src/+/HEAD/styleguide/c++/c++.md)
- [ ] Test plan is specified in PR before merging

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on
- [ ] All relevant documentation has been updated, for instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protections
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-Compatibility-Exceptions-in-Brave
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide
  - [ ] https://github.com/brave/brave-browser/wiki/P3A

### CODE REVIEW

File: CHANGELOG_DESKTOP.md [hunk 1]
```diff
@@ -769,7 +769,7 @@
  - Removed Brave autoplay exceptions. ([#9597](https://github.com/brave/brave-browser/issues/9597))
  - Removed unnecessary balance API calls from Brave Rewards. ([#9132](https://github.com/brave/brave-browser/issues/9132))
  - Removed retry logic for auto-contribute if auto-contribute or Brave Rewards is disabled. ([#10257](https://github.com/brave/brave-browser/issues/10257))
- - Removed known user tracking parameters from Doubleclick query strings. ([#9879](https://github.com/brave/brave-browser/issues/9879))
+ - Removed known user tracking parameters from Double click query strings. ([#9879](https://github.com/brave/brave-browser/issues/9879))
```
Comment (bsclifton) [inline CHANGELOG_DESKTOP.md:772]: This one is not a typo; it's the name of a company actually. Although, the proper capitalization is `DoubleClick` if you wanted to fix

**Comment by bsclifton** [comment]
Left a comment there on the 2nd one; can you also do a rebase and squash the commits together?

**Comment by MalikIdreesHasanKhan** [comment]
@bsclifton, Did as you asked :)

**Comment by bsclifton** [comment]
@MalikIdreesHasanKhan perfect! One last thing - can you drop the change (or addition) for the `.replit` file? So then there will just be one change to `CHANGELOG_DESKTOP.md` 😄

**Comment by MalikIdreesHasanKhan** [comment]
Done @bsclifton

**Comment by MalikIdreesHasanKhan** [comment]
Hi, I am reopening this PR as I am having difficulties rebasing

**Pull request closed without merging**